README.md

```diff
@@ -81,7 +81,12 @@ tba
 <a id="install_linux"></a>
 
 ## linux
-tba
+
+An example with the LADSPA plugin illustrates how to install any of the other formats: copy to either the system library in $PREFIX/lib/ladspa with $PREFIX being /usr/local or /usr.
+For a user-only install copy into $HOME/.ladspa/ or if ~/.local is preferred, declare `export LADSPA_PATH=~/.local/lib/ladspa` in your ~/.profile and restart your user session.
+This scheme applies equally to clap, lv2, vst and vst3 too.
+
+Want to use a plugin within a particular software or tool? Ardour loads VST3. Audacity VST, LADSPA and LV2 plugins. ffmpeg and sox can use LADSPA.
```

Comment:
This line is a bit odd, Ardour supports LADSPA, LV2, VST2 and VST3 plugins. The way you phrased makes it seems like Audacity supports these but Ardour does not.